Makefile

```diff
@@ -468,12 +482,22 @@ release: ## Re-generate generated code and run tests
 	$(MAKE) --no-print-directory test
 
 build: ## Build all packages and all tests (ensures everything compiles)
-	$Q echo 'Building all packages...'
+	$Q echo 'Building all packages and submodules...'
 	$Q go build ./...
+	$Q cd common/archiver/gcloud; go build ./...
```

Comment:
I assume we're doign a build here to just ensure everything compiles? There's no like weird linking step or anythign required I assume?

Reply:
yea, this is just handy for building _literally everything_, since `go build .` or `go build cmd/server` would only build packages that the server (transitively) imports, which is not actually everything.

and because the "build all tests" tactic is not all that easy to remember or type.  and `go build ./...` does not build tests.